docs/data/charts/styling/styling.md

```diff
@@ -42,6 +42,67 @@ Here is an example of the d3 Categorical color palette.
 
 {{"demo": "ColorTemplate.js"}}
 
+### Values color
+
+Colors can also be set according to item values.
+To do so, use the `colorMap` property of the corresponding axis.
+
+For demonstration, you can refer to the chart specific pages:
+
+- [bar chart](/x/react-charts/bars/#color-scale)
+- [line chart](/x/react-charts/lines/#color-scale)
+- [scatter chart](/x/react-charts/scatter/#color-scale)
+
+The `colorMap` property can accept three kind of objects defined bellow
+
+#### Piecewise color map
+
+The piecewise configuration takes an array of N `thresholds` values, and N+1 `colors`.
+
+```ts
+{
+  type: 'piecewise';
+  thresholds: V[];
```

Comment:
What was the rational behind those two arrays instead of a single `{ threshold: V, color: string }[]`?

And `V` should be renamed `Value` IMHO, it took me a few seconds to understand what it meant.

Reply:
Because threshold should contain one less values than the colors

```
blue <y=0> red
```
translate in 
```
trheshold=[0]
colors=['blue', 'red']
```

Another option more user friendly is to do  something like `[{ "<": 0, color: 'blue' }, { ">=": 0, color: 'red' }]`

It was my first try, but you have to babysit the dev to make sure they did not overlap ranges, and manage the notion of strict. equal inequalities. So I removed it and whent to an interface closer to the d3 ones

Reply:
> but you have to babysit the dev to make sure they did not overlap ranges

Yeah, not a great DX and too error prone